components/resources/brave_vpn_strings.grdp

```diff
@@ -43,7 +43,7 @@
   <message name="IDS_BRAVE_VPN_UNABLE_CONNECT_INFO" desc="Description of which region VPN couldn't connect to"><ph name="PRODUCT_NAME">$1<ex>Brave Firewall + VPN</ex></ph> couldn't connect to the <ph name="REGION_NAME">$2<ex>Tokyo</ex></ph> server. You can try again, or choose another</message>
 
   <message name="IDS_BRAVE_VPN_BUY" desc="Button title that includes the price of the product">
-    Buy for <ph name="PRODUCE_PRICE">$1<ex>$4.99/mo</ex></ph>
+    Buy <ph name="PRODUCE_PRICE">$1<ex>$4.99/mo</ex></ph>
```

Comment:
Would we need to keep the placeholder? Decision seemed to be to remove the price permanently as we won't know it at this point, and leave the string at `"Buy"`